pkgs/tools/misc/heimdall/default.nix

```diff
@@ -51,17 +49,16 @@ mkDerivation rec {
       mkdir -p $out/{bin,share/doc/heimdall,lib/udev/rules.d}
       install -m755 -t $out/bin                bin/*
       install -m644 -t $out/lib/udev/rules.d   ../heimdall/60-heimdall.rules
-      install -m644 ../Linux/README   $out/share/doc/heimdall/README.linux
-      install -m644 ../OSX/README.txt $out/share/doc/heimdall/README.osx
+      install -m644 ../README.md               $out/share/doc/heimdall/README.md
+      install -m644 ../doc/*                   $out/share/doc/heimdall/
     '';
 
   meta = with lib; {
     broken = stdenv.hostPlatform.isDarwin;
-    homepage = "http://www.glassechidna.com.au/products/heimdall/";
```

Comment:
Removed this primarily because the new upstream removed the remaining donation links already.
The homepage links in the source code are still intact, but I feel like having (outdated) homepage links in the source code is significantly different from us advertising the link to the user.